src/Microsoft.CodeAnalysis.Analyzers/Core/AnalyzerDiagnosticCategory.cs

```diff
@@ -1,12 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
FYI: Large number of file changes here are due to the fact that our category strings and resources were duplicated and spread across projects. Now all of them have been unified into the AnalyzerUtilities project.